keras_cv/models/segmentation/deeplab.py

```diff
@@ -64,89 +76,89 @@ def __init__(
                 )
             self._backbone_passed = backbone
             if backbone == "resnet50_v2":
-                backbone = keras_cv.models.ResNet50V2(
-                    include_rescaling=include_rescaling, include_top=False
+                backbone = ResNetV2(
+                    stackwise_filters=BACKBONE_CONFIG["ResNet50V2"][
+                        "stackwise_filters"
+                    ],
+                    stackwise_blocks=BACKBONE_CONFIG["ResNet50V2"]["stackwise_blocks"],
+                    stackwise_strides=BACKBONE_CONFIG["ResNet50V2"][
+                        "stackwise_strides"
+                    ],
+                    stackwise_dilations=BACKBONE_CONFIG["ResNet50V2"][
```

Comment:
Should stackwise_dilations be defined for all ResNets, or only for this particular application?

It seems like this should break our weights -- is that correct?

Reply:
It shouldn't break our weights, i.e., the current classification models will not use this dilation, but deeplab models will use it but still load from classification model pretrained weights